Documentation/tutorial/links_and_cross_references.md

```diff
@@ -267,3 +267,23 @@ When creating documentation for your own .NET library, it is needed to link to t
 This can be downloaded on [nuget](https://www.nuget.org/packages/msdn.4.5.2).
 
 > You can also manually create `xrefmap.yml` for project that is not generated by DocFX to be able to cross reference topics in it.
+
+## Advance: more options for cross reference
+
+You can create a cross link with following options:
+
+- `text`: the display text when the cross reference is has resolved correctly.
```

Comment:
and line 281

---
In reply to: [83810721](https://github.com/dotnet/docfx/pull/796#discussion_r83810721) [](ancestors = 83810721)